plugin/src/main/java/org/opensearch/ml/action/memorycontainer/TransportCreateMemoryContainerAction.java

```diff
@@ -149,49 +149,67 @@ private MLMemoryContainer buildMemoryContainer(MLCreateMemoryContainerInput inpu
             .build();
     }
 
-    private void createMemoryDataIndices(
-        MLMemoryContainer container,
-        User user,
-        ActionListener<String> listener
-    ) {
+    private void createMemoryDataIndices(MLMemoryContainer container, User user, ActionListener<String> listener) {
         String userId = user != null ? user.getName() : "default";
         MemoryConfiguration configuration = container.getConfiguration();
         String indexPrefix = configuration != null ? configuration.getIndexPrefix() : null;
 
         // Convert to lowercase as OpenSearch doesn't support uppercase in index names
         final String sessionIndexName = indexPrefix.toLowerCase(Locale.ROOT) + "-session";
-        final String shortTermMemoryIndexName = indexPrefix.toLowerCase(Locale.ROOT) + "-short-term-memory";
+        final String workingMemoryIndexName = indexPrefix.toLowerCase(Locale.ROOT) + "-working-memory";
```

Comment:
Change to `final String sessionIndexName = configuration.getSessionIndexName();` to keep consistent

Same for line 158, 160